scripts/exgfs_atmos_fbwind.sh

```diff
@@ -14,15 +14,15 @@
 # echo "         Nov 2019 - B Vuong  Removed WINTEMV bulletin (retired)"
 #####################################################################
 
-source "${USHgfs}/preamble.sh"
+source "${HOMEgfs}/ush/preamble.sh"
 
-cd $DATA
+cd "${DATA}" || exit 2
 
 ######################
 # Set up Here Files.
 ######################
 
-job_name=$(echo $job|sed 's/[jpt]gfs/gfs/')
+job_name="${job/[jpt]gfs/gfs}"
```

Comment:
This changes the output filename from tran.fbwnd_pacific.**gfs_atmos_fbwind_CC** to tran.fbwnd_pacific.**fbwind**, missing the cycle information. Is it intentionally?

Reply:
It shouldn't. This line should function identically to what was here before.

Reply:
The output filename on Line 71 is tran.fbwnd_pacific.**${job_name}**. Did `$job` change?

Reply:
Not exactly. When it is run in ops, `$job` comes from the scheduler jobname. I don't know what that is. In development, we set it in the rocoto pre-job stub (`jobs/rocoto/fbwind.sh`). This is the only job I know of that includes the job name in a filename (which seems to me to be a poor practice for this very reason). Presumably, if this were run in ops, the filename would still have the cycle in it.

Reply:
I'll change this so it no longer relies on `$job` for the filename.

Reply:
`${job_name}` is only used to define output filename hereafter. If it's not used elsewhere, we can simply define:

`job_name=${model}_${COMPONENT}_${cyc}`

or something similar to meet EE2 standard.

Reply:
Trying to come up with a new name. Do you know what "tran" means in the current filename?

Reply:
No, I don't.

Reply:
Would `${RUN}.atmos.t${cyc}z.wind_profile.pacific.ascii` be correct?
CC @aerorahul

Reply:
Actually, I'm just going to make it `${RUN}.atmos.t${cyc}z.fbwind.pacific.ascii`, at least for now. No need to worry about it if it winds up getting ditched anyway.